tests/conftest.py

```diff
@@ -126,6 +126,7 @@ def mock_session(
     """
     from otelib.settings import Settings
 
+    # For now I need python backend to run test for real
```

Comment:
What do you mean?

Reply:
Dead comment. I was trying something here and forgot to remove the comment